README.md

```diff
@@ -233,7 +233,7 @@ Expecto supports the following test constructors:
 All of the above compile to a `Test` value that you can compose. For example,
 you can compose a `test` and a `testCaseAsync` in a `testList` which you wrap in
 `testSequenced` because all tests in the list use either `Expect.fasterThan` or
-they are using `Expecto.BenchmarkDotNet` for performance tests.
+they are using `Expecto.BenchmarkDotNet` for performance tests. You have to remember that the full name of test(s) should be unique across your test project.
```

Comment:
Linebreak at 80 chars

Reply:
Just make this line a new paragraph and we're done. You can embolden the text **the fully qualified names of the tests need to be unique across your test project**.